README.md

```diff
@@ -122,20 +122,21 @@ Comments
 **Example:**
 ```objc
 /*
- * This is a multi-line comment. The opening and closing markers are on their
- * own lines, and each other line is preceded by a * that is indented by one
- * space.
- *
- * This is a new paragraph in the same block comment.
+ This is a multi-line comment. The opening and closing markers are on their
+ own lines, and each other line is preceded by a * that is indented by one
```

Comment:
Remove “, and each other line is preceded by a * that is indented by one space.” Since its no longer true